### ISSUE TITLE
Some docstring convention for Reporting Variables

### ISSUE THREAD

**Issue by Loisel** [pik-piam/remind2#41]
It would be great to have some inline documentation of reported variables, that can be read via script for output creation. Something along the lines:
```
...
#§ Liquid fuels from Biomass required in the transport sector
    setNames((dimSums(mselect(vm_demFeSector,all_enty1=c("fepet","fedie"),all_enty="seliqbio",emi_sectors="trans")  ,dim=3,na.rm=T)), "FE|Transport|Liquids|+|Biomass (EJ/yr)"),
...
```
In this case, the explanation is trivial, but it is not for some (especially emission) variables. This
- makes the documentation of changes easier for the developer, who changes the definition while he changes the code
- can be checked when reviewing PRs
- can be used with, e.g., the IAMC template which requires a definition of the variables

**Comment by Renato-Rodrigues** [comment]
I agree, but I think that any solution should be done in a way that this description is also easily saved in a variables metadata file. 
csv file example:
```
variable;unit;description
FE|Transport|Liquids|+|Biomass";"EJ/yr";Liquid fuels from Biomass required in the transport sector
```

I would guess the RSE group is the most competent one to define the best way to support this.
